demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementTagController.php

```diff
@@ -317,8 +318,12 @@ public function tagListEditAction(
         // Check if we need to import tags
         if (\array_key_exists('r_import', $requestPost)) {
             if (\array_key_exists('r_importCsv', $requestPost) && '' != $requestPost['r_importCsv']) {
-                $statementHandler->importTags($procedure, $requestPost['r_importCsv']);
-                $this->getMessageBag()->add('confirm', 'explanation.import.topicsAndTags');
+                try {
+                    $statementHandler->importTags($procedure, $requestPost['r_importCsv']);
+                    $this->getMessageBag()->add('confirm', 'explanation.import.topicsAndTags');
+                } catch (DuplicatedTagTitleException $e) {
```

Comment:
Same as above

Reply:
there are here other throwed exceptions which have to be handled, but BadRequestException i think is not one of them. all exceptions are anyway catched and handled now here https://github.com/demos-europe/demosplan-core/commit/5e0e05c33fbcc40c4b1965e8b4845441083c9eb3